resources.go

```diff
@@ -122,7 +122,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		CSharp: &tfbridge.CSharpInfo{
 			PackageReferences: map[string]string{
-				"Pulumi":                       "1.5.0-*",
+				"Pulumi":                       "1.7.0-preview",
```

Comment:
Should we be referencing the latest version (`1.8.1-preview` now)? I don't see a great reason to keep all packages consistently pointing to 1.7.0.

Reply:
Let's keep it on 1.7.0 to keep it consistent with AWS et al